### PR TITLE
Try to get ipympl working

### PR DESCRIPTION
**Pull request by mpmdean**
I am proposing to allow the use of ipympl widgets as raised in  #955 

I thought these simple additional packages and build commands would suffice, but somehow it does work. This PR functions correctly within jupyter notebooks, but it does not work with jupyterlab. The cell only responds by returning ‘Loading widget…’ , but it never loads anything. 

Do not merge this!

### CODE REVIEW

File: scipy-notebook/Dockerfile [hunk 1]
```diff
@@ -50,7 +50,7 @@ RUN conda install --quiet --yes \
     # Also activate ipywidgets extension for JupyterLab
     # Check this URL for most recent compatibilities
     # https://github.com/jupyter-widgets/ipywidgets/tree/master/packages/jupyterlab-manager
-    jupyter labextension install @jupyter-widgets/jupyterlab-manager@^1.0.01 --no-build && \
```
Comment (romainx) [inline scipy-notebook/Dockerfile]: Updated to `@jupyter-widgets/jupyterlab-manager@^1.1.0`

File: scipy-notebook/Dockerfile [hunk 2]
```diff
@@ -26,6 +26,7 @@ RUN conda install --quiet --yes \
     'h5py=2.10.*' \
     'hdf5=1.10.*' \
     'ipywidgets=7.5.*' \
+    'ipympl=0.3.*'\
```
Comment (romainx) [inline scipy-notebook/Dockerfile]: Replaced by `ipympl=0.5.*`

Comment (mpmdean) [reply scipy-notebook/Dockerfile]: Thanks! I've independently verified that this indeed solves it.

**Comment by mpmdean** [comment]
Thanks to @romainx this is now working. 

I would propose that  the status:Needs Debugging label is removed and that the pull request can be considered for merging.

**Comment by romainx** [comment]
Great thank you. I forgot a point in my review. 
Could you also update the [documentation](https://jupyter-docker-stacks.readthedocs.io/en/latest/using/selecting.html) to mention ipyml?

**Comment by mpmdean** [comment]
@romainx I've now added mention of ipympl to the selecting an image section of the docs.

**Comment by romainx** [comment]
@mpmdean thank you, everything seems to be fine and ready to merge now 👍.

**Comment by parente** [comment]
Thanks for the PR @mpmdean and for reviewing @romainx. I'm merging master into this branch to make sure there are no build errors before merging back to master.

**Comment by mpmdean** [comment]
@romainx

I am still seeing problems with ipympl in jupyterlab (see the image below). The issue is the same one discussed previously. I've reproduced this both by pulling from dockerhub and by building the current version. I'm stuck as to what happened, as it was working before and I cannot see what change is correlated with the problem.

Screen Shot 2020-03-09 at 2 55 25 PM

**Comment by romainx** [comment]
@mpmdean You're right same result on my side and same lack of explanation ... :expressionless:
I keep you posted if I found something.

**Comment by mpmdean** [comment]
@romainx I tried to backtrack to a working version via:

1. git reset --hard 4bff6ab1978f682ab089baa261ce0361190e411d (which is right after I thought I had a working version and before the ubuntu update you mention)
2. build the base notebook
3. edit the tag of the scipy-notebook to that build and then rebuild.

This failed in the way we are discussing.

**Comment by romainx** [comment]
I will try to investigate tonight. In the meantime you could open an issue in their project since it's easily reproducible.

**Comment by mpmdean** [comment]
I've open an issue.

**Pull request merged**